### DISCUSSION TITLE
I think I'll start using MILESTONES

### DISCUSSION THREAD

**Discussion by togfoxy** [togfoxy/MarsLander#179] [Ideas]
I've always got my eyes on iterative improvements including the releases that go out to fans and players. When I see some genuine improvements that the players can see and appreciate then I like to push a new release.

There are lots of ISSUES in GitHub (which is great) with competing priorities and different levels of interest. I think I'm going to use github milestones to identify the next release and to assign issues to it. When that milestone is reached then a release will be pushed.

We don't have to work on just that milestone. We can keep doing what we're doing but me tracking milestones will let me know when a release is ready.

**Comment by MadByteDE** [comment]
It's a good idea. Might mean releases will be irregular but that shouldn't be an issue.

**Reply by togfoxy** [reply]
We don't get paid so irregular releases is a given.  :)

**Reply by Philbywhizz** [reply]
Wait? we get paid? :)

**Reply by MadByteDE** [reply]
Wait, shouldn't we get paid? :stuck_out_tongue_winking_eye:

**Comment by togfoxy** [comment]
Nobody call the union!

**Comment by togfoxy** [comment]
I've nominated v0.12 milestone to be about screen resolution and mplayer polish. There is still no clear vision on game modes so this will be 'spawn - fly as far as you can - reset - go again' mode.

Objective = out distance your mplayer buddies.
